### DISCUSSION TITLE
MN launch tool - log4j feature uses incorrect library

### DISCUSSION THREAD

**Discussion by waveskimmer** [micronaut-projects/micronaut-core#11200] [General]
Launch tool uses the old library for the earlier (<2.0) slf4j version: 

    runtimeOnly("org.apache.logging.log4j:log4j-slf4j-impl")

where as: 

    runtimeOnly("org.apache.logging.log4j:log4j-slf4j2-impl")

is correct.   (Note the extra 2 in slf4j2).  

```
applicationType: default
defaultPackage: com.example
testFramework: junit
sourceLanguage: kotlin
buildTool: gradle_kotlin
features: [app-name, gradle, http-client-test, junit, kotlin, kotlin-application, ksp, log4j2, micronaut-aot, micronaut-build, micronaut-http-validation, netty-server, properties, readme, serialization-jackson, shade, static-resources]
```

Seems like a bug, but where do I report it?

**Comment by nedelva** [comment]
@waveskimmer Why do you insist on using Log4j2 and not use Logback instead, like the rest of the cool kids here 😄
